src/client/app/+home/home.component.ts

```diff
@@ -1,12 +1,12 @@
 import { FORM_DIRECTIVES } from '@angular/common';
 import { Component } from '@angular/core';
 
-import { NameListService } from '../../shared/index';
+import { NameListService } from '../shared/index';
 
 @Component({
   selector: 'sd-home',
-  templateUrl: 'app/+home/components/home.component.html',
-  styleUrls: ['app/+home/components/home.component.css'],
+  templateUrl: 'app/+home/home.component.html',
```

Comment:
Hey!
As a matter of fact
`moduleId = module.id` was in the code of the seed some while ago but got removed at some point. I'll have a look for the specific commit which removed it to see if there was any reason for this. Would the inclusion also enable the use of the barrel imports without the `/index` suffix, or would there have to be another configuration step in the `tsconfig.json` as you mentioned earlier?